main.py

```diff
@@ -9,22 +9,24 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
-# Dynamic mapping Settings
-dm_datastream_url = 'https://raw.githubusercontent.com/elastic/elasticsearch/4211c3ae051c51df3e649ed687a4639aed66e794/x-pack/plugin/core/src/main/resources/ecs-data-stream-mappings.json'
-dm_ecs_url = 'https://raw.githubusercontent.com/elastic/elasticsearch/4211c3ae051c51df3e649ed687a4639aed66e794/x-pack/plugin/core/src/main/resources/ecs-dynamic-mappings.json'
-dm_default_url = 'https://raw.githubusercontent.com/elastic/elasticsearch/4211c3ae051c51df3e649ed687a4639aed66e794/x-pack/plugin/core/src/main/resources/string-to-keyword-dynamic-mapping.json'
+# Dynamic ECS mappings
+dm_ecs_url = 'https://raw.githubusercontent.com/elastic/elasticsearch/13fb93511c23fe0d1a02de07644a0415253010c5/x-pack/plugin/core/src/main/resources/ecs-dynamic-mappings.json'
+
 # ECS Settings
 ecs_url = 'https://raw.githubusercontent.com/elastic/ecs/main/generated/ecs/ecs_flat.yml'
 
 files = ['field_mappings.json', 'dynamic_template.json',
          'ecs_generated.json', 'ecs_flat.json', 'results.json']
 # Elasticsearch Settings
-es_host = "https://localhost:9200"
+es_host = "http://localhost:9200"
 es_index = "testindex5"
 es_user = "elastic"
-es_pass = "changeme"
+es_pass = "password"
```

Comment:
I figure the more common use case is to use with Elasticsearch's default, which is probably also what we will use for the actual test, right?